.github/workflows/build.yml

```diff
@@ -0,0 +1,326 @@
+            echo $resourcesToDelete
+
+            az resource delete --verbose --ids $resourcesToDelete
+            az disk delete --yes --resource-group ${{ env.testResourceGroup }} --ids ${unentitledOsDiskId} ${unentitledDataDiskIds} ${entitledOsDiskId} ${entitledDataDiskIds}
+      - name: Generate SAS url
+        id: generate-sas-blob-url
+        run: |
+          #Get a minus-24-hour date for the SAS token
+          minus24HoursUtc=$(date -u --date "$dte -24 hour" +%Y-%m-%dT%H:%MZ)
+          echo $minus24HoursUtc
+
+          #Get a plus-30-day date for the SAS token
+          plus30DaysUtc=$(date -u --date "$dte 30 day" +%Y-%m-%dT%H:%MZ)
+          echo $plus30DaysUtc
+
+          vhdStorageAccountAccessKey=$(az storage account keys list --account-name ${{ env.vhdStorageAccountName }} --query "[?keyName=='key1'].value" -o tsv)
+          echo ${vhdStorageAccountAccessKey}
+
+          sasTokenForOffer=$(az storage container generate-sas --connection-string "DefaultEndpointsProtocol=https;AccountName=${{ env.vhdStorageAccountName }};AccountKey=${vhdStorageAccountAccessKey};EndpointSuffix=core.windows.net" --name vhds --permissions rl --start "${minus24HoursUtc}" --expiry "${plus30DaysUtc}" -o tsv)
+          echo $sasTokenForOffer
+
+          blobStorageEndpoint=$( az storage account show -n ${{ env.vhdStorageAccountName }} -g ${{ env.testResourceGroup }} -o json | jq -r '.primaryEndpoints.blob' )
+          echo $blobStorageEndpoint
+
+          osDiskSasUrl=${blobStorageEndpoint}vhds/${{ env.vmName }}.vhd?$sasTokenForOffer
+          echo "osDiskSasUrl: ${osDiskSasUrl}"
+          dataDiskSasUrl=${blobStorageEndpoint}vhds/${{ env.vmName }}datadisk1.vhd?$sasTokenForOffer
+          echo "dataDiskSasUrl: ${dataDiskSasUrl}"
+          
+          echo "osDiskSasUrl: ${osDiskSasUrl}, dataDiskSasUrl: ${dataDiskSasUrl}" > sas-url.txt
+          sasUrls=`cat sas-url.txt`
+          echo "sas-url: ${sasUrls}"
+      - name: Upload sas-url.txt
+        uses: actions/upload-artifact@v2
+        with:
+          name: sasurl
+          path: sas-url.txt
+  summary:
+    needs: build
+    if: always()
+    runs-on: ubuntu-latest
+    steps:
+      - name: Download sas-url.txt
+        uses: actions/download-artifact@v2
+        with:
+          name: sasurl
+      - name: summarize jobs
+        if: ${{!(github.event_name == 'schedule' && github.repository_owner != 'WASdev')}}
+        run: |
+            workflow_jobs=$(curl -H "Accept: application/vnd.github.v3+json" https://api.github.com/repos/WASdev/azure.websphere-traditional.image/actions/runs/${{ github.run_id }}/jobs)
+
+            success_build_job=$(echo $workflow_jobs | jq '.jobs | map(select(.name=="build" and .conclusion=="success")) | length')
+            echo "$success_build_job"
+            if (($success_build_job == 0));then
+                echo "Job failed, send notification to Teams"
+                curl ${{ secrets.MSTEAMS_WEBHOOK }} \
```

Comment:
Please add MSTEAMS_WEBHOOK to env section

Reply:
Done